client/post-editor/editor-mobile-navigation/index.jsx

```diff
@@ -23,17 +57,26 @@ const EditorMobileNavigation = React.createClass( {
 
 		return (
 			<div className="editor-mobile-navigation">
-				<Site indicator={ false } site={ this.props.site } />
-				<button className="button editor-mobile-navigation__toggle" onClick={ this.toggleSidebar }>
-					{ this.translate( 'Actions' ) }
-				</button>
-				<Gridicon
-					icon="cross"
-					onClick={ this.props.onClose }
-					className="editor-mobile-navigation__close" />
+				<div className="editor-mobile-navigation__actions">
+					<Gridicon
+						icon="chevron-left"
+						onClick={ this.props.onClose }
```

Comment:
Done, changed the Gridicons to buttons.

The `screen-reader-text` seems like an improvement to be made in other PR, though :)